starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,52 @@
 // Names and Input
+var hacker1 = "Bob";
+
+console.log("The drivers name is " + hacker1);
+var hacker2 = prompt("What is the navigators name?");
+console.log("The navigators name is " + hacker2);
 
 
 //Conditionals
 
+function longestName(){
+  var driverLength = hacker1.length;
+  var navigatorLength = hacker2.length;
+  if (driverLength > navigatorLength){
+    console.log("The Driver has the longest name, it has " + driverLength + " characters");
+  } else if (driverLength < navigatorLength) {
+    console.log("Yo, navigator got the longest name, it has " +  navigatorLength  + "  characters");
+  } else {
+    console.log("wow, you both got equally long names, " + navigatorLength +  "characters!!");
+  }
+}
+
+function driverLettersCaps(){
+  var result = "";
+  for (var i = 0; hacker1.length > i  ;i++ ){
+    result += hacker1[i].toUpperCase();
+    result += " ";
+  }
+  console.log(capSize);
```

Comment:
Is capSize defined?

Reply:
Changed the var name halfway through, forgot to edit the function call.